ci/ci.sh

```diff
@@ -162,6 +162,25 @@ test_python() {
       -python/ray/tests:test_k8s_operator_unit_tests
       -python/ray/tests:test_tracing  # tracing not enabled on windows
       -python/ray/tests:kuberay/test_autoscaling_e2e # irrelevant on windows
+      -python/ray/tests:test_runtime_env.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_2.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_working_dir.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_working_dir_2.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_working_dir_3.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_working_dir_remote_uri.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_conda_and_pip.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_conda_and_pip_2.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_conda_and_pip_3.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_conda_and_pip_4.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_conda_and_pip_5.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_complicated.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_validation.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_ray_minimal.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_env_vars.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_packaging.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_plugin.py  # see test_windows_runtime_env
+      -python/ray/tests:test_runtime_env_fork_process.py  # see test_windows_runtime_env
+      
```

Comment:
This might be helpful: https://github.com/ray-project/ray/blob/b02029b29f77adf546d7907f7ef1bbbf8973f095/.buildkite/pipeline.yml#L28

We're using a tag `post_wheel_build` to identify which tests we need to run.  Does it make sense to use something like this here?  That way we don't have to copy all the runtime env tests here (possibly including ones that don't require a wheel), and it might be more maintainable to stick with using this tag.